src/main/java/com/synopsys/integration/blackduck/imageinspector/containerfilesystem/pkgmgr/apk/ApkDbInfoFileParser.java

```diff
@@ -0,0 +1,57 @@
+package com.synopsys.integration.blackduck.imageinspector.containerfilesystem.pkgmgr.apk;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.synopsys.integration.blackduck.imageinspector.containerfilesystem.pkgmgr.pkgmgrdb.DbRelationshipInfo;
+
+public class ApkDbInfoFileParser {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public DbRelationshipInfo parseDbRelationshipInfoFromFile(File dbInfoFile) {
```

Comment:
Very minor, but might be good, in new code at least, to follow the Detect convention and have parsers take in List<String> that the caller reads from the file. (I guess Jordan is getting to me.)